watch-package.js

```diff
@@ -76,8 +78,9 @@ module.exports = function watchPackage (pkgDir, exit) {
     var proc = processes[script] = spawn(nodemon, args, {
       env: process.env,
       cwd: pkgDir,
-      stdio: 'pipe'
+      stdio: inherit === true? 'inherit' : 'pipe'
```

Comment:
Can you test on a non-windows machine if you do this instead:
`stdio: inherit === true? ['pipe', 'inherit', 'pipe'] : 'pipe'`?
On windows inheriting the entire stdio causes any subsequent input to be overlaid on the previous output.